admin_guide/persistent_storage/pod_security_context.adoc

```diff
@@ -0,0 +1,285 @@
+  securityContext:
+    fsGroup: 1234
+  volumes:
+    - name: ebs-volume
+      awsElasticBlockStore:
+      volumeID: <VOLUME ID>
+----
+====
+
+The above pod will attach the specified AWS EBS volume to the node,
+format it if needed, mount it, and connect it to the pod. It will then
+apply the FSGroup to it as detailed above.
+
+Currently the list of volumes which support ownership Management includes:
+
+* AWS Elastic Block.
+* Cinder.
+* GCE Persistent Disk.
+* ISCSI.
+* Empty Dir.
+* Ceph RBD.
+* Git Repo.
+
+GlusterFS, and NFS do not support ownership management.
+
+== SELinuxOptions
+
+The pod security context allows you to specify SELinux labels with
+which to run containers in your pod. Additionally volumes which
+support SELinux management will be relabeled so that they are
+accessible by the specified label, and- depending on how exclusionary
+the label is- only that label.
+
+In effect this means two things:
+
+* If the container is unprivileged the volume will be given a `type`
+  which is accessible by unprivileged containers. Usually
+  `svirt_sandbox_file_t`.
+* If a `level` is specified the volume will be labeled with the given
+  MCS label.
+
+[NOTE]
+====
+Level and MCS label are used interchangeably in this document.
+====
+
+For your volume to be accessible by your pod the pod must have both
+categories of the volume. So a pod with `s0:c1,c2` will be able to
+access volumes with `s0,c1,c2`, and a volume with `s0` will be
+accessible by all pods.
+
+[WARNING]
+====
+Hardcoding MCS labels into your pod definition makes it easy for
+others to determine what MCS label is needed to access the same volume
+as the defined pod. So it is especially important to rely on the MCS
+labels allocated by OpenShift and not use this option with care.
+====
+
+SELinux options are specified as follows:
+
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: ebs-web
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+      volumeMounts:
+          - name: ebs-volume
+            mountPath: "/usr/share/nginx/html"
+  securityContext:
+    seLinuxOptions:
+    level: "s0:c123,c456"
+  volumes:
+    - name: ebs-volume
+      awsElasticBlockStore:
+      volumeID: <VOLUME ID>
+----
+====
+
+Currently the list of volumes which support SELinux Management includes:
+
+* AWS Elastic Block.
+* Cinder.
+* GCE Persistent Disk.
+* ISCSI.
+* Empty Dir.
+* Ceph RBD.
+* Git Repo.
+
+GlusterFS, and NFS do not support SELinux management.
+
+== UID and GID Management with NFS and GlusterFS
+
+As mentioned above NFS and GlusterFS do not support ownership
+management. This is because they do not allow `chown` and `chmod` on
+the client side. As a result, when you are using NFS and GlusterFS you
+will need to set the appropriet ownership on the server side then use
+`supplementalGroups` to match the group. You can also use `runAsUser`
+to match the user id.
```

Comment:
`setgid` is not required. It might even interfere with an application which wants to own its own files.